index.html

```diff
@@ -29,7 +29,7 @@
    <script id="book-template" type="text/x-handlebars-template">
    {{#each this}}
      <div class="book">
-       <img src="http://covers.openlibrary.org/b/isbn/{{isbn}}-M.jpg" alt={{name}} class="cover">
+       <img src="http://covers.openlibrary.org/b/isbn/{{isbn}}-M.jpg" alt="default.jpg" class="cover">
```

Comment:
`alt` doesn't work as your expecting it to work, the `alt` attribute is for alternative text.